llama-index-integrations/readers/llama-index-readers-google/llama_index/readers/google/drive/base.py

```diff
@@ -57,68 +55,41 @@ def __init__(
             },
         }
 
-    def _get_credentials(self) -> Tuple[Credentials, GoogleDrive]:
+    def _get_credentials(self) -> Tuple[Credentials]:
         """Authenticate with Google and save credentials.
-        Download the credentials.json file with these instructions: https://developers.google.com/drive/api/v3/quickstart/python.
-            Copy credentials.json file and rename it to client_secrets.json file which will be used by pydrive for downloading files.
-            So, we need two files:
-                1. credentials.json
-                2. client_secrets.json
-            Both 1, 2 are essentially same but needed with two different names according to google-api-python-client, google-auth-httplib2, google-auth-oauthlib and pydrive libraries.
+        Download the service_account_key.json file with these instructions: https://cloud.google.com/iam/docs/keys-create-delete.
 
         Returns:
-            credentials, pydrive object.
+            credentials
         """
         from google_auth_oauthlib.flow import InstalledAppFlow
-        from pydrive.auth import GoogleAuth
 
         # First, we need the Google API credentials for the app
         creds = None
 
-        if Path(self.token_path).exists():
-            creds = Credentials.from_authorized_user_file(self.token_path, SCOPES)
-        elif Path(self.credentials_path).exists():
-            creds = service_account.Credentials.from_service_account_file(
-                self.credentials_path, scopes=SCOPES
```

Comment:
this didn't make sense. overloading the credentials_path argument to refer to both client_secrets.json as described [here](https://developers.google.com/identity/protocols/oauth2/web-server) and service account keys as described [here](https://cloud.google.com/iam/docs/keys-create-delete)

Reply:
technically still a breaking change in functionality here, but should be ok?